src/test/java/com/google/api/codegen/gapic/testdata/php/php_library.baseline

```diff
@@ -1896,7 +1896,7 @@ function sampleGetShelf()
     $stringBuilder = new StringBuilder();
 
     try {
-        $response = $libraryServiceClient->getShelf($formattedName, $options);
+        $response = $libraryServiceClient->getShelf($formattedName, $options, ['message' => $message, 'stringBuilder' => $stringBuilder]);
```

Comment:
ditto

Reply:
same as above.

Reply:
This change is necessary. This is quite similar to Ruby: we were not using the correct `initCode` in the template. We actually constructed the variable `$stringBuilder` a few lines above so it should be passed in.